app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -240,26 +244,36 @@ def build_metric_id(type, resource, metric_id)
     # selected target server. This server is identified
     # by ems_ref, which in Hawkular terms is the
     # fully qualified resource path from Hawkular inventory
-    def run_generic_operation(operation, ems_ref, parameters = {})
-      with_provider_connection do |connection|
+    def run_generic_operation(dmr_operation, ems_ref, parameters = {})
         the_operation = {
-          :operationName => operation,
+          :operationName => dmr_operation,
```

Comment:
Left a link here that explains this better: https://issues.jboss.org/browse/HWKAGENT-120?focusedCommentId=13270161&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13270161